src/layouts/SubPageNav.vue

```diff
@@ -21,52 +33,51 @@ export default {
         ? '4'
         : '0'
     "
-    height="80"
+    height="auto"
     color="appBackground"
+    class="pb-3"
     style="
     position: fixed;
     width: 100%;
-    z-index: 5;"
+    z-index: 4;"
   >
     <v-row
       no-gutters
-      class="d-flex align-center mx-auto pb-3"
+      class="d-flex align-center mx-auto"
+      :class="{
+        'justify-center': $vuetify.breakpoint.smAndDown
+      }"
       style="max-width: 1440px;"
     >
-      <span
-        style="height: 100%;
-        position: absolute;"
-        ><slot name="page-type"> </slot
-      ></span>
-      <v-col :sm="$slots['page-actions'] ? 6 : 12" style="padding-left: 50px;">
-        <v-row>
-          <v-col
-            cols="12"
-            class="d-flex align-center justify-space-between pb-1"
-          >
-            <div style="width: inherit;">
-              <div v-if="$vuetify.breakpoint.mdAndUp">
-                <span v-if="$slots.breadcrumbs" style="font-size: 0.875rem;"
-                  ><slot name="breadcrumbs"></slot
-                ></span>
-              </div>
-              <div class="headline"
-                ><slot name="page-title"></slot
-                ><slot
-                  v-if="$slots.breadcrumbs && $vuetify.breakpoint.smAndDown"
-                  name="breadcrumbs"
-                ></slot>
-              </div>
-            </div>
-          </v-col>
-        </v-row>
+      <v-col
+        :sm="$slots['page-actions'] ? 6 : 12"
+        class="d-flex align-end"
+        :class="{}"
```

Comment:
out of curiosity, what's the reason for this?

Reply:
That was accidentally left from testing some reactive classes, removed now!